.github/workflows/ci.yml

```diff
@@ -54,20 +55,7 @@ jobs:
           run: |
             cd backend
             make test
-
-    lint:
-      name: Run linter
-      runs-on: ubuntu-latest
-
-      steps:
-        - name: Checkout repository
-          uses: actions/checkout@v4
-
-        - name: Setup Go
-          uses: actions/setup-go@v5
-          with:
-            go-version: '1.22.5'
-        
+          
         - name: Install golangci-lint
```

Comment:
not sure if it is better (at least, seems more verbose) and if it actually works, but may be use https://github.com/golangci/golangci-lint-action instead?

Reply:
changed to golangci-lint-action